### PR TITLE
Update portal.json

### PR DESCRIPTION
**Pull request by fochoaog**
Change to proper spanish syntax of a few fields.

Got some code for us? Awesome 🎊!

Please include a description of your change & check your PR against this list, thanks!

- [x] There's a clear use-case for this code change, explained below
- [ ] Commit message has a short title & references relevant issues
- [ ] The build will pass (run `yarn test:all` and `yarn lint`)

We appreciate your contribution!

### CODE REVIEW

File: ghost/i18n/locales/es/portal.json [hunk 1]
```diff
@@ -6,63 +6,63 @@
...
     "Cancel subscription": "Cancelar suscripción",
     "Cancellation reason": "Razón de cancelación",
     "Change": "Cambiar",
     "Check spam & promotions folders": "Comprueba la carpeta de spam y promociones",
-    "Check with your mail provider": "Comprueba con tu proveedor de correo",
+    "Check with your mail provider": "Comprueba con tu proveedor de correo electrónico",
     "Choose": "Elige",
     "Choose a different plan": "Elige un plan diferente",
-    "Choose your newsletters": "Elige tus boletines",
+    "Choose your newsletters": "Elige tus boletínes",
```
Comment (cathysarisky) [inline ghost/i18n/locales/es/portal.json:31]: is this a regional difference?  I'm getting i (not í) as the correct spelling for the plural...

File: ghost/i18n/locales/es/portal.json [hunk 2]
```diff
@@ -6,63 +6,63 @@
...
-    "Email preferences": "Preferencias",
+    "Email preferences": "Preferencias de correo electrónico",
     "Emails": "Correos electrónicos",
     "Emails disabled": "Correos electrónicos desactivados",
-    "Ends {{offerEndDate}}": "Termina el {{offerEndDate}}",
+    "Ends {{offerEndDate}}": "Termina él {{offerEndDate}}",
     "Error": "Error",
-    "Expires {{expiryDate}}": "Expira el {{expiryDate}}",
-    "Forever": "para siempre",
+    "Expires {{expiryDate}}": "Expira él {{expiryDate}}",
```
Comment (cathysarisky) [inline ghost/i18n/locales/es/portal.json:57]: Is this (él) correct?

**Review by cathysarisky — COMMENTED** [review]
I left a few comments - it looks like there are some good fixes here, but I'm confused by boletínes and  él - are these spelling errors, or is this a regional spelling difference?  Thanks!

File: ghost/i18n/locales/es/portal.json (hunk 2 above)
Comment (fochoa8) [reply ghost/i18n/locales/es/portal.json:57]: Just rechecked, El is indeed without accent. Sorry about that, consulted pronouns and the actual dictionary and definitions on the RAE dictionary. "Expira el {{expiryDate}}" like it is written.

File: ghost/i18n/locales/es/portal.json (hunk 1 above)
Comment (fochoa8) [reply ghost/i18n/locales/es/portal.json:31]: Wrong there too. Boletín is only with accent, not in plural, which is only boletines.

File: ghost/i18n/locales/es/portal.json (hunk 2 above)
Comment (fochoa8) [reply ghost/i18n/locales/es/portal.json:57]: Those are the only two that are indeed wrong, the rest is fine.

Comment (cathysarisky) [reply ghost/i18n/locales/es/portal.json:57]: Thanks for confirming. If you could please update the PR, I'll be glad to mark it approved with your fixes.

Comment (fochoa8) [reply ghost/i18n/locales/es/portal.json:57]: Sure,

Comment (fochoa8) [reply ghost/i18n/locales/es/portal.json:57]: [portal.json](https://github.com/user-attachments/files/17136930/portal.json)

That's the file changed.

Comment (cathysarisky) [reply ghost/i18n/locales/es/portal.json:57]: OK, so what you need to do is go back into the editor on this branch (fochoaog:patch-2), make the edits (follow the tutorial if needed), and then commit them.  That should automatically update the PR.

**Comment by fochoa8** [comment]
Did quite an extensive review since You asked and finally did proof reading to all the portal.json file, it is checked against dictionary, syntaxis and what are technical terms in IT Science, but from a point of view that Is understandable to the end user. More than 1 hour of proof reading and translating and explaining in fewer words what needs to be known.

[portal.json](https://github.com/user-attachments/files/17137916/portal.json)

Will do the full edit in brief.

**Comment by cathysarisky** [comment]
Thanks so much! I appreciate your work.  Did you see my message about updating the PR, rather than attaching a file, please?

  While I've got you, there are a couple of spots with desabilitado -- should those  be deshabilitado ?

**Comment by fochoa8** [comment]
Yes, wait a bit until I merge it fully. Should not take much.

Thanks.

**Comment by fochoa8** [comment]
This is my commit with said info. https://github.com/fochoa8/Ghost/commit/612c549ed23daf1b9a4197e57e6b795c81e1e20f

**Comment by fochoa8** [comment]
Changed of repository, if You can fix it that way, it would be great.

**Comment by fochoa8** [comment]
> Thanks so much! I appreciate your work. Did you see my message about updating the PR, rather than attaching a file, please?
> 
> While I've got you, there are a couple of spots with desabilitado -- should those be deshabilitado ?

[https://github.com/TryGhost/Ghost/compare/main...fochoa8:Ghost-1:patch-1](https://github.com/TryGhost/Ghost/compare/main...fochoa8:Ghost-1:patch-1)

Follow that url. It has the full file translation.

**Comment by fochoa8** [comment]
> Thanks so much! I appreciate your work. Did you see my message about updating the PR, rather than attaching a file, please?
> 
> While I've got you, there are a couple of spots with desabilitado -- should those be deshabilitado ?

Here is the pull request:

[https://github.com/TryGhost/Ghost/pull/21127](https://github.com/TryGhost/Ghost/pull/21127)

**Comment by 9larsons** [comment]
Closing this out as there's multiple other PRs submitted now with overlap.

**Pull request closed without merging**